src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -186,6 +186,10 @@ WARNING: If the combination between RAM and CORES does not match any existing AW
 
     -   *spotPrice:* The maximum price that you are willing to pay per hour per instance (your bid price). Amazon EC2 Spot instances allow you to bid on spare Amazon EC2 computing capacity. Since Spot instances are often available at a discount compared to On-Demand pricing. If your bid price is greater than the current Spot price for the specified instance, and the specified instance is available, your request is fulfilled immediately. Otherwise, the request is fulfilled whenever the Spot price falls below your bid price or the specified instance becomes available. Spot instances run until you terminate them or until Amazon EC2 must terminate them (also known as a Spot instance interruption). More information available on https://aws.amazon.com/ec2/spot/
 
+    -   *securityGroupNames:* The securityGroupNames option allows you to specify the name of the Security groups configured as a virtual firewalls (to control inbound and outbound traffic) for the EC2 instances hosting the proactive nodes. More information regarding Amazon EC2 Security Group available on http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html
```

Comment:
i think links should be added like the following : 
http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-network-security.html[title of the link^].